java_gen/pre-written/src/main/java/org/projectfloodlight/openflow/types/MacAddress.java

```diff
@@ -20,6 +21,9 @@
  */
 
 public class MacAddress implements OFValueType<MacAddress> {
+
+    private static final Pattern MAC_ADDRESS_PATTERN = Pattern.compile("^([0-9A-Fa-f]{2}[:-]){5}([0-9Aa-f-F]{2})$");
```

Comment:
Oh interesting, didn't know that.

IMHO, still better to enforce consistency...